jobs/scripts/i18n-find-untranslated.sh

```diff
@@ -1,7 +1,13 @@
 #!/bin/sh
 set -x
 
-pip install -r requirements.txt
+PYTHON_VERSION="{py}"
+
+uv python install ${{PYTHON_VERSION}}
```

Comment:
Actually you do not need to install it.
`uv venv -p ${PYTHON_VERSION}` 
is enough and install the requested Python version.